src/test/java/core/basesyntax/HelloWorldTest.java

```diff
@@ -1,7 +1,206 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
+import core.basesyntax.dao.DataConverter;
+import core.basesyntax.dao.DataConverterImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.models.FruitTransaction;
+import core.basesyntax.models.activities.ActivityHandler;
+import core.basesyntax.models.activities.BalanceActivityHandler;
+import core.basesyntax.models.activities.PurchaseActivityHandler;
+import core.basesyntax.models.activities.ReturnActivityHandler;
+import core.basesyntax.models.activities.SupplyActivityHandler;
+import core.basesyntax.services.DataProcessor;
+import core.basesyntax.services.DataProcessorImpl;
+import core.basesyntax.services.ReportGenerator;
+import core.basesyntax.services.ReportGeneratorImpl;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Test;
+
 public class HelloWorldTest {
```

Comment:
Your tests should have the same structure as that main solution. Create separate classes.